src/utils/eventAccessibilityLinter.ts

```
@@ -0,0 +1,127 @@
+import { parse } from 'node-html-parser';
```

ESLint

[error] Replace `'node-html-parser'` with `"node-html-parser"`

(prettier/prettier)